--- gwenview-1.1.3/desktopfiles/konqgwenview.desktop~	2004-07-24 00:09:57.000000000 +0000
+++ gwenview-1.1.3/desktopfiles/konqgwenview.desktop	2004-07-24 00:09:57.000000000 +0000
@@ -24,7 +24,6 @@
 Name[sk]=Prehliadať s Gwenview
 Name[sl]=Brskaj z Gwenview
 Name[sr]=Прегледај помоћу Gwenview-а
-Name[sr@Latn]=Прегледај помоћу Gwenview-а
 Name[sv]=Bläddra med Gwenview
 Name[ta]=GWEN பார்வையை வைத்து தேடு
 Name[uk]=Перегляд у Gwenview
--- gwenview-1.1.3/desktopfiles/gwenview.desktop~	2004-07-24 00:09:39.000000000 +0000
+++ gwenview-1.1.3/desktopfiles/gwenview.desktop	2004-07-24 00:09:39.000000000 +0000
@@ -24,7 +24,6 @@
 GenericName[ru]=Программа просмотра изображений
 GenericName[sl]=Pregledovalnik slik
 GenericName[sr]=Прегледник слика
-GenericName[sr@Latn]=Прегледник слика
 GenericName[ss]=Sibuki semfanekiso
 GenericName[sv]=Bildvisare
 GenericName[ta]=அழைப்பவர் அடையளம் சிறு நிரல்
@@ -55,7 +54,6 @@
 Comment[sk]=Jednoduchý prehliadač obrázkov
 Comment[sl]=Preprost pregledovalnik slik
 Comment[sr]=Једноставан приказивач слика
-Comment[sr@Latn]=Једноставан приказивач слика
 Comment[sv]=En enkel bildvisare
 Comment[ta]=ஒரு சாதாரண உருவ பார்வையாளர்
 Comment[tr]=Basit bir resim izleyici
